libraries/AP_RangeFinder/AP_RangeFinder_PWM.cpp

```diff
@@ -67,7 +67,7 @@ bool AP_RangeFinder_PWM::get_reading(uint16_t &reading_cm)
     if (value_us == 0 || sample_count == 0) {
         return false;
     }
-    reading_cm = (value_us/sample_count) * 1e-1f; // correct for LidarLite.  Parameter needed?
+    reading_cm = value_us/(sample_count * 10); // correct for LidarLite.  Parameter needed?  Converts from m -> cm here
```

Comment:
Alright, I know I'm one of those Americans who use weird units. But uh, dividing by 10 is not converting meters to centimeters, which makes the new comment realllllly confusing?